tests/unit/local/apigw/test_local_apigw_service.py

```diff
@@ -918,15 +917,20 @@ def test_lambda_auth_unauthorized_response(
         service_response_mock = Mock(return_value=make_response_mock)
         self.api_service.service_response = service_response_mock
 
-        request_mock.return_value = ("test", "test")
-
         mock_context = {"key": "value"}
         invoke_mock.side_effect = [{"context": mock_context}, Mock()]
 
         unauth_mock = Mock()
         service_err_mock.lambda_authorizer_unauthorized.return_value = unauth_mock
 
-        result = self.api_service._request_handler()
+        flask_app = flask.Flask("test_flask_app")
+
+        with flask_app.test_request_context() as context:
+            context.request.method = "test"
+            context.request.path = "/test"
+
+            result = self.api_service._request_handler()
+
```

Comment:
To address the deprecated call (the werkzeug call happens within the `flask_app.test_request_context()` like mentioned), we should be able to patch the call to `_create_method_arn` since we aren't concerned/testing the output of this method.

The reason why `werkzeug` is no longer pinned is to resolve a CVE. We could also upgrade Flask, however this may have other cascading side effects.

Reply:
Done.